torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -587,6 +589,20 @@ def metadata_fn(*primals):
     )
 
 
+class PropagateUnbackedSymInts(torch.fx.Interpreter):
+    def run_node(self, n: torch.fx.Node):
+        import sympy
+        from torch.fx.experimental.symbolic_shapes import free_unbacked_symbols
+
+        result = super().run_node(n)
+        # TODO: handle Tensor returns
+        if 'example_value' in n.meta:
+            if isinstance(result, torch.SymInt):
+                torch._check(result == n.meta['example_value'])
```

Comment:
don't we only have to do this check if result has any unbacked symbols in it? (or is it harmless to do this check when it's backed, since presumably it would no-op)

Reply:
Yeah, is harmless if it's backed